cmake/DAGMC_macros.cmake

```diff
@@ -69,6 +72,16 @@ macro (dagmc_setup_options)
     set(BUILD_GEANT4 ON)
     set(BUILD_FLUKA  ON)
   endif ()
+
+  if (BUILD_SHARED_LIBS AND NOT BUILD_STATIC_LIBS)
+    set(BUILD_STATIC_EXE OFF)
```

Comment:
Probably not, but we'd need to have a discussion about where to install them. It's convenient to install both types of libraries because they have different filenames (`.so` vs. `.a`), but the executables have the same name.

Reply:
Do we need both sets of executables?  Seems like we need both libraries but not 2 sets of executables